modules/common/profiles/host_hardening.nix

```diff
@@ -0,0 +1,47 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+#
```

Comment:
should this file should be called hardening.nix? and not host-hardening? Because it includes settings for both guest and host

Reply:
Changed to `hardening.nix`